LAMMPS-GUI Improvements (#4651)
lammps/lammps: doc/src/Howto_lammps_gui.rst

----

> LAMMPS-GUI tries to provide an experience similar to what people

Reviewer: ```suggestion
LAMMPS-GUI aims to provide the traditional experience of running LAMMPS: 
```

> traditionally would have when running LAMMPS using a text editor, the

Reviewer: ```suggestion
using a text editor, a
```

> command-line window, and launching the LAMMPS executable but just rolled

Reviewer: ```suggestion
command-line window, and launching the LAMMPS executable, but integrated
```

> into a single executable:

Reviewer: May be a single "application" instead of a single "executable" ?

> - writing & editing LAMMPS input files with a text editor

Reviewer: ```suggestion
- Write and edit LAMMPS input files using a built-in text editor.
```

> - run LAMMPS on those input file with selected command-line flags

Reviewer: ```suggestion
- Run LAMMPS on these input files with selectable command-line flags.
```

> - extract data from the created files and visualize it with

Reviewer: Extract data from the generated files for visualization using external software.

  external software

That traditional procedure is effective for people proficient in using the

> command-line, as that allows them to use tools for the individual steps

Reviewer: ```suggestion
command-line, as it allows them to use tools for the individual steps
```

data from the generated output.  It also integrates well with graphical
desktop environments where the `.lmp` filename extension can be
registered with LAMMPS-GUI as the executable to launch when double
clicking on such files using a graphical file manager.  LAMMPS-GUI also

> has support for drag-n-drop for opening inputs, i.e.  an input file can

Reviewer: ```suggestion
supports drag-and-drop: an input file can
```

> be selected and then moved and dropped on the LAMMPS-GUI executable, and

Reviewer: ```suggestion
be dragged onto the LAMMPS-GUI executable, which will then
```

> LAMMPS-GUI will launch and read the file into its buffer.  In many cases

Reviewer: ```suggestion
launch and load the file into its buffer.  In many cases,
```

LAMMPS-GUI will be integrated into the graphical desktop environment and

> can be launched like other applications.

Reviewer: ```suggestion
can be launched just like any other applications.
```

LAMMPS-GUI thus makes it easier for beginners to get started running

> LAMMPS simulations since you can start with learning LAMMPS right away.

Reviewer: ```suggestion
LAMMPS simulations, as they can focus on learning LAMMPS right away.
```

> It is therefore very suitable for tutorials on LAMMPS, since you only

Reviewer: ```suggestion
It is particularly well-suited for LAMMPS tutorials, since users only
```

> need to learn how to use a single program that is available as

Reviewer: ```suggestion
need to work with a single, ready-to-use program,
```

> ready-to-use, pre-compiled packages for popular operating systems

Reviewer: ```suggestion
available as pre-compiled packages for popular operating systems
```

(Linux, macOS, Windows), for most of the tasks.  This saves time and

> people can focus on learning LAMMPS instead of having to compile LAMMPS,

Reviewer: ```suggestion
allows users to focus on learning LAMMPS itself, without needing to compile it first,
```

> learn how to use the command line and a text editor.

Reviewer: ```suggestion
learn how to use the command line, or learn to use a separate text editor.
```

The tutorials at https://lammpstutorials.github.io/ are specifically
updated for use with LAMMPS-GUI and their tutorial materials can be
downloaded and edited directly from within the GUI while automatically
loading the matching tutorial instructions into a webbrowser.

Yet the basic control flow remains similar to running LAMMPS from the

> command line and thus the barrier for replacing part of the

Reviewer: ```suggestion
command line, so the barrier to replacing parts of the
```

functionality of LAMMPS-GUI with external tools is low.  That said,

> LAMMPS-GUI has some unique functionality that is not easily found

Reviewer: ```suggestion
LAMMPS-GUI offers some unique features that are not easily found
```
